README.md

```diff
@@ -0,0 +1,37 @@
+# GwardiaHub Project 🌐
+
+## 📄 About the project
+
+We want to create website with Django, that will be useful to as many as three different user groups!
+
+- Our classmates
+- @Gwardia-Czapli/czlonek-gwardii,
```

Comment:
Could you please add a link to this team, or make it bold? It would enhance the readability. It doesn't look quite good now...

Reply:
Maybe i should type
`[Gwardia Czapli](https://github.com/Gwardia-Czapli) members` 
instead of linking to team, because why do this in the first place?

Reply:
That sounds like a good idea to me!